index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h4>Hello world we are developers</h4>
+
+
+<h1 id="title">Survey form </h1>
+  <p id="description"> Lorem ipsum dolor sit amet consectetur
+  </p>
+
+<form id="survey-form">
+  <label for="name" id="name-label">Name</label>
+    <input id="name" placeholder="name" required>
+    <label for="email" id="email-label">Email</label>
+    <input type="email" id="email" placeholder="email" required>
+    <label for="number" id="number-label">Number</label>
+    <input type="number" id="number" min="7" max="15" required placeholder="number">
+    <select id="dropdown" name="dropdown">
```

Comment:
Please add a title attribute to your select element by adding a label to it

Reply:
Kindly rephrase this with the reason why you asked for this. Like "Kindly add a title attribute to your select element by adding a label to it which will clearly define what it is used for."